styles/styles.css

```diff
@@ -0,0 +1,100 @@
+/* Footer */
+
+footer {
+    background-color: white;
+}
+
+.footer-container {
+    width: 100%;
+    padding: 70px 0px 20px;
+}
+
+.brand-logo {
+    width: 200px;
```

Comment:
For responsive design, an image should generally not have a fixed width unless it's specifically intended for a certain device or context. Instead, consider using relative units like percentages (e.g., width: 20%) to make the image scale dynamically based on its parents size.